src/ui/public/vis/agg_configs.js

```diff
@@ -152,7 +152,7 @@ class AggConfigs extends IndexedArray {
 
     if (hierarchical) {
       // collect all metrics, and filter out the ones that we won't be copying
-      nestedMetrics = _(this.bySchemaGroup.metrics)
+      nestedMetrics = _(this.byTypeType.metrics)
```

Comment:
we should not use schema names here, as we might not have schema defined on aggconfig at this point